publisher/publisher.go

```diff
@@ -246,9 +253,19 @@ func (pub *Impl) SubmitToSingleCTWithResult(ctx context.Context, req *pubpb.Requ
 		return nil, err
 	}
 
-	isPrecert := req.Precert
+	// TODO(#7161): Simply check req.Kind using IsAnyNilOrZero once it is always
+	// being set by the RA. Note that this block is not capable of producing
+	// pubpb.SubmissionType_info, which matches prior behavior.
```

Comment:
Until the multi-step deploy is completed we'll be unable to see submission results to informational logs?

Reply:
Today, all submissions show up in our metrics as either "precert" or "final". For the one week between this change and the next change, all submissions will show up as either "sct" or "final". After the next change, the metric points showing up as "sct" will get correctly split into "sct" and "info".